pepper-apis/src/main/java/org/broadinstitute/ddp/route/PatchFormAnswersRoute.java

```diff
@@ -493,6 +505,25 @@ private DateAnswer convertDateAnswer(String stableId, String guid, String actIns
         }
     }
 
+    private FileAnswer convertFileAnswer(Handle handle, Response response, String stableId, String guid,
+                                         String instanceGuid, JsonElement value) {
+        boolean isNull = (value == null || value.isJsonNull());
+        if (isNull || (value.isJsonPrimitive() && value.getAsJsonPrimitive().isString())) {
```

Comment:
Since we allow PATCH-ing `null`, there's a few implications which we didn't touch on in the API discussions:

* The first PATCH call to a file question can pass in `null`, which will result in a new FileAnswer being created with a `null` as `value`.
* Client can get back a FileAnswer object with a valid `answerGuid` but `null` for `value`. I think this behavior is okay. It more-or-less means the user has answered the question before but has not "cleared" their answer value. Client can use the `answerGuid` to PATCH again with a new file upload.

```
// Something like this:
"question": {
   ...
   answers: [
    { "answerGuid": "XYZ", "value": null, ... }
  ]
}
```